workflow-dev/tpl/deis-controller-deployment.yaml

```diff
@@ -56,6 +56,12 @@ spec:
 {{- end}}
 {{- end}}
           env:
+            # NOTE(jchauncey): Possible values are:
+            # enabled - allows for open registration
+            # disabled - turns off open registration
+            # admin_only - allows for registration by an admin only.
+            - name: REGISTRATION_MODE
+              value: enabled
```

Comment:
can we have the value taken from the params.toml because that is where people add all the configuration options and has more visibility.